src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -229,7 +230,9 @@ public void Start()
             this.LoadKeysLookupLock();
 
             // Find the last chain block received by the wallet manager.
-            this.WalletTipHash = this.LastReceivedBlockHash();
+            HashHeightPair hashHeightPair = this.LastReceivedBlockHash();
+            this.WalletTipHash = hashHeightPair.Hash;
```

Comment:
Why not remove both of them and replace with single HashHeightPair variable?

Reply:
That will effect many tests, lots more files to change.